lib/net/http/header.rb

```diff
@@ -126,46 +126,59 @@
 # Various convenience methods retrieve values, set values, query values,
 # set form values, or iterate over fields.
 #
+# === Setters
+#
+# \Method #[]= can set any field, but does little to validate
+# to validate the new value;
+# most of the other setter methods provides some validation:
+#
+# - #[]=: Sets the string or array value for the given key.
+# - #add_field: Creates or adds to the array value for the given kry.
```

Comment:
Technically, `#add_field` is also like `#[]=` in that it doesn't validate the value.

Reply:
I've weakened the validation claim.